guides/v2.0/config-guide/bootstrap/magento-bootstrap.md

```diff
@@ -14,19 +14,19 @@ functional_areas:
 
 ## Overview of bootstrapping   {#config-boot-overview}
 
-To run the Magento application, the following actions are implemented in <a href="{{ site.mage2000url }}index.php" target="_blank">index.php</a>:
+To run the Magento application, the following actions are implemented in [index.php]({{ site.mage2000url }}index.php){: target="_blank"}:
 
-*	Include <a href="{{ site.mage2000url }}app/bootstrap.php" target="_blank">app/bootstrap.php</a> which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
-*	Create an instance of <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a>. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
-*	Create a Magento application instance (<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/AppInterface.php" target="_blank">\Magento\Framework\AppInterface</a>).
+*	Include [app/bootstrap.php]({{ site.mage2000url }}app/bootstrap.php){: target="_blank"} which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
+*	Create an instance of [\\Magento\\Framework\\App\\Bootstrap]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php){: target="_blank"}. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
```

Comment:
I'm going to stop my review here. Please review all of the changes for double `\` and the errant space between `:` and `target` throughout.

Reply:
I'll pick up my review when those changes have been made.